packages/components/src/date-time/stories/index.js

```diff
@@ -70,3 +70,17 @@ export const WithDaysHighlighted = () => {
 		/>
 	);
 };
+
+export const WithInvalidDates = () => {
```

Comment:
In tandem with #35665 and #35744, we're in the process of making our Storybook work more as standalone documentation ([example end result](https://wordpress.github.io/gutenberg/?path=/docs/components-experimental-unitcontrol--default)). With that end goal in mind, we want to make sure that any stories we add are useful and understandable as part of public documentation.

It's a bit unclear to me what this story is demonstrating, so I think it warrants a JSDoc comment to describe what it's trying to show (the JSDoc comment will show up in the Storybook Docs view). And if it's actually just a test fixture, we can remove the story.

Reply:
Ooh that's nice! I'll add a JSDoc comment. This story just demonstrates how to use the `isInvalidDate` prop. Definitely this component's stories could do with a larger update to be more coherent down the line.